client/__init__.py

```diff
@@ -0,0 +1,265 @@
+from .definitions import *
```

Comment:
shall we include the `requirements.txt`

Reply:
Good question - added packaging/shipping as a TODO